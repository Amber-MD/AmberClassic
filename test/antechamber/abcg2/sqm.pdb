ATOM      1  C1  MOL     1       3.545   1.434  -0.002  1.00  0.00           C
ATOM      2  H1  MOL     1       2.509   1.244  -0.391  1.00  0.00           H
ATOM      3  H2  MOL     1       4.245   0.645  -0.391  1.00  0.00           H
ATOM      4  H3  MOL     1       3.536   1.406   1.120  1.00  0.00           H
ATOM      5  N1  MOL     1       4.006   2.771  -0.463  1.00  0.00           N
ATOM      6  H4  MOL     1       4.006   2.771  -1.493  1.00  0.00           H
ATOM      7  C2  MOL     1       3.079   3.839  -0.002  1.00  0.00           C
ATOM      8  H5  MOL     1       3.432   4.832  -0.390  1.00  0.00           H
ATOM      9  H6  MOL     1       2.046   3.627  -0.390  1.00  0.00           H
ATOM     10  H7  MOL     1       3.059   3.861   1.121  1.00  0.00           H
ATOM     11  C3  MOL     1       5.395   3.041  -0.002  1.00  0.00           C
ATOM     12  H8  MOL     1       5.728   4.041  -0.390  1.00  0.00           H
ATOM     13  H9  MOL     1       6.078   2.238  -0.391  1.00  0.00           H
ATOM     14  H10 MOL     1       5.423   3.046   1.121  1.00  0.00           H
