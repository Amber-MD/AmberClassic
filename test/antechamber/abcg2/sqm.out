            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 13

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       26
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.000886 seconds
| QMMM:                 Dspev diag routine = 0.008657 seconds
| QMMM:                Dspevd diag routine = 0.000736 seconds
| QMMM:                Dspevx diag routine = 0.003803 seconds
| QMMM:                 Dsyev diag routine = 0.000586 seconds
| QMMM:                Dsyevd diag routine = 0.000704 seconds
| QMMM:                Dsyevr diag routine = 0.001065 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.000191 seconds
| QMMM:
| QMMM: Using dsyev routine (diag_routine=5).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        3.5400    1.4200    0.0000
  QMMM:     2        2      H        2.5460    1.2490   -0.3830
  QMMM:     3        3      H        4.2190    0.6730   -0.3830
  QMMM:     4        4      H        3.5320    1.3950    1.0800
  QMMM:     5        5      N        4.0060    2.7710   -0.4390
  QMMM:     6        6      H        4.0060    2.7710   -1.4480
  QMMM:     7        7      C        3.0680    3.8500    0.0010
  QMMM:     8        8      H        3.4180    4.7960   -0.3830
  QMMM:     9        9      H        2.0830    3.6360   -0.3830
  QMMM:    10       10      H        3.0510    3.8700    1.0800
  QMMM:    11       11      C        5.4100    3.0440    0.0010
  QMMM:    12       12      H        5.7170    4.0040   -0.3830
  QMMM:    13       13      H        6.0550    2.2680   -0.3830
  QMMM:    14       14      H        5.4360    3.0480    1.0800

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10      151.8954 kcal/mol        0.0006 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -45.6739  -34.0997  -34.0996  -29.9914  -24.5021  -22.3491  -22.3491  -18.9851
 -18.9851  -18.4014  -18.3879  -18.3878  -18.1784   -4.2950   -2.1938   -2.1860
  -2.1860   -1.1868   -1.1868   -1.0552   -0.9374   -0.9374   -0.7561   -0.6909
  -0.6113   -0.6113

 Heat of formation   =        151.89538254 kcal/mol  (        6.58667805 eV)

 Total SCF energy    =     -16649.04868556 kcal/mol  (     -721.95692665 eV)
 Electronic energy   =     -63734.51812625 kcal/mol  (    -2763.73609671 eV)
 Core-core repulsion =      47085.46944069 kcal/mol  (     2041.77917006 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.196
     2      H                  0.148
     3      H                  0.148
     4      H                  0.149
     5      N                  0.003
     6      H                  0.251
     7      C                 -0.196
     8      H                  0.148
     9      H                  0.148
    10      H                  0.149
    11      C                 -0.196
    12      H                  0.148
    13      H                  0.148
    14      H                  0.149
 Total Mulliken Charge =       1.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    19.243   13.310   -1.416   23.440
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        3.5454    1.4338   -0.0021
  QMMM:     2        2      H        2.5087    1.2438   -0.3908
  QMMM:     3        3      H        4.2451    0.6455   -0.3907
  QMMM:     4        4      H        3.5358    1.4062    1.1204
  QMMM:     5        5      N        4.0062    2.7711   -0.4629
  QMMM:     6        6      H        4.0062    2.7713   -1.4934
  QMMM:     7        7      C        3.0785    3.8388   -0.0017
  QMMM:     8        8      H        3.4324    4.8317   -0.3902
  QMMM:     9        9      H        2.0460    3.6271   -0.3904
  QMMM:    10       10      H        3.0595    3.8605    1.1208
  QMMM:    11       11      C        5.3947    3.0406   -0.0018
  QMMM:    12       12      H        5.7276    4.0408   -0.3902
  QMMM:    13       13      H        6.0777    2.2379   -0.3907
  QMMM:    14       14      H        5.4232    3.0458    1.1207

           --------- Calculation Completed ----------

